cfg/config.yaml

```diff
@@ -25,7 +25,6 @@ master:
       - "hyperkube apiserver"
       - "hyperkube kube-apiserver"
       - "apiserver"
-      - "openshift start master api"
```

Comment:
For my own learning: Why are these removed?

Reply:
Basically, these values are specific to the OpenShift, so they belong in the **cfg/rh-0.7/config.yaml** instead.